control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2094,6 +2133,17 @@ components:
         - replicas
         - size
         - thin
+    CreateSnapshotBody:
+      example:
+        parameters: {"param1": "value1"}
+      description: Create Snapshot Body
+      type: object
+      properties:
```

Comment:
Does it mean that we pass input body as an opaque key:value object without specifying its exact properties ?

Reply:
No, this is as per the csi specs, the CreateSnapshotRequest has, 3 fields the snapshot name, the source and the parameters. We get the first 2 from url and thus I have pur the parameters in the body. `parameters` is basically one of the fields in the payload.

Reply:
We can remove it but, since its part of csi-spec it must have some significance that's why I kept it,

Reply:
Same as the create volume, they're opaque parameters from a class, but in this case from the snapshot class rather than the storage class AFAIK.
We may want to "de-opaque" some parameters to specific arguments like we do with create volume.

Reply:
That's right. So shall I remove it for now?

Reply:
Yes I think so, we can always add it later